integrations/marketo-v2/lib/index.js

```diff
@@ -98,17 +103,50 @@ var Marketo = (module.exports = integration('Marketo V2')
 
 Marketo.prototype.initialize = function() {
   var self = this;
```

Comment:
I don't think you need to reassign `this` to `self` here. When passed as an argument to a method (as on line 108), `this` should still refer to the outer context (in this case, the `initialize` prototype method).

Reply:
Wouldn't that only apply if you are trying to access `this` itself inside of the function? All I'm doing on 108 is to pass a reference to the value inside the options object. Options become available via the scope of the function (`this`).  

I'll move the reasign below the variable definitions but please let me know what you think.